cmd/utils/flags.go

```diff
@@ -360,6 +360,11 @@ var (
 		Name:  "extradata",
 		Usage: "Block extra data set by the miner (default = client version, deprecated, use --miner.extradata)",
 	}
+	MinerRecommitIntervalFlag = cli.IntFlag{
+		Name:  "miner.recommit",
+		Usage: "Sealing work recommit interval(ms), will be dynamically adjusted by the system if it is too short",
```

Comment:
Lets shorted this description a bit so it doesn't wrap on smaller width terminals. Eg.

`Time interval (ms) to recreate the block being mined.`

I don't think there's a need to mention the automatic adjustments. That's a protective measure, the user can't change it anyway, so lets keep the help cleaner/clearer.

Reply:
Also, I think we should rather use a `cli.DurationFlag` instead. Those support nicer formats, like `--miner.recommit=1.5s`